src/models/stores/serial.ts

```diff
@@ -0,0 +1,98 @@
+import { NodeChannelInfo } from "src/plugins/dataflow-tool/model/utilities/node";
+
+export class SerialDevice {
+    value: string;
+    localBuffer: string;
+    private port: any; // SERIAL TODO: types SerialPort | null 
```

Comment:
```suggestion
    private port: SerialPort | null;
```
I tried pulling this down locally and it just worked. When we tried this together I think it may have been something simple like needing to reload the VSCode window to get it to re-parse the `tsconfig.json` (where we added the reference to the types).